README.md

```diff
@@ -18,121 +18,23 @@ __Cloudformation:__ Cloudformation will create the CI/CD components utilizing:
 - __CloudWatch__
 - __GitHub__.
 
-The intended behavior is to pull the framework from GitHub, and the custom configuration for a specific event source from its CodeCommit repository. The Cloudformation template has the potential to be separately deployed for various event sources, while the only difference will be the event source's CodeCommit repository that contains the custom configuration data for that event source. This allows us to maintain a generic framework that is event source agnostic and relieved of sensitive data. 
+The intention is to have a template of a framework which we store in GitHub, however, the custom configuration for each specific event source can be stored in  either a CodeCommit or separate GitHubrepository. The Cloudformation template has the potential to be separately deployed for various event sources, where the difference between each deployed stack will be the event source's configuration stored in the CodeCommit/GitHub repository. This allows us to maintain a generic framework that is event source agnostic and relieved of sensitive data. 
```

Comment:
Typo "GitHubrepository" in the end of first sentence.

Reply:
Thanks! Fixed :)